src/features/organization/queries/getOrganizationRoleInfo.ts

```diff
@@ -0,0 +1,56 @@
+import { queryOptions } from '@tanstack/react-query';
+import { apiClient } from '@/config/apiClient';
+import { SchemaRole } from '@/lib/api.gen';
+import { queryKeys } from '@/react-query/constants';
+
+export interface GetOrganizationRoleInfoResponse extends SchemaRole {
```

Comment:
Can you put this type in the patch file instead?

Reply:
For sure!